Makefile

```diff
@@ -0,0 +1,166 @@
+	$(EXEC) $(CONSOLE) doctrine:schema:validate
+
+
+##
+## Assets
+##---------------------------------------------------------------------------
+
+watch: node_modules                                                                                    ## Watch the assets and build their development version on change
+	$(EXEC) yarn watch
+
+assets: node_modules                                                                                   ## Build the development version of the assets
+	$(EXEC) yarn dev
+
+assets-build: node_modules                                                                             ## Build the production version of the assets
+	$(EXEC) yarn build
+
+
+##
+## Tests
+##---------------------------------------------------------------------------
+
+tests:                                                                                                 ## Run all the PHP tests
+	$(EXEC) bin/phpunit
+
+lint: lint-symfony php-cs                                                                              ## Run lint on Twig, YAML, XLIFF, and PHP files
+
+lint-symfony: lint-yaml lint-twig lint-xliff                                                           ## Lint Symfony (Twig and YAML) files
+
+lint-yaml:                                                                                             ## Lint YAML files
+	$(EXEC) $(CONSOLE) lint:yaml config
+
+lint-twig:                                                                                             ## Lint Twig files
+	$(EXEC) $(CONSOLE) lint:twig templates
+
+lint-xliff:                                                                                            ## Lint Translation files
+	$(EXEC) $(CONSOLE) lint:xliff translations
+
+php-cs: vendor                                                                                         ## Lint PHP code
+	$(PHPCSFIXER) fix --diff --dry-run --no-interaction -v
+
+php-cs-fix: vendor                                                                                     ## Fix PHP code to follow the convention
+	$(PHPCSFIXER) fix
+
+security-check: vendor                                                                                 ## Check for vulnerable dependencies
+	$(EXEC) vendor/bin/security-checker security:check
+
+test-schema: vendor                                                                                    ## Test the doctrine Schema
+	$(EXEC) $(CONSOLE) doctrine:schema:validate --skip-sync -vvv --no-interaction
+
+test-all: lint test-schema security-check tests                                                        ## Lint all, run schema and security check, then unit and functionnal tests
+
+
+##
+## Dependencies
+##---------------------------------------------------------------------------
+
+deps: vendor assets                                                                                    ## Install the project dependencies
+
+
```

Comment:
What do you think do we need to add command to update dependancies?
`deps-update:
## Update the project dependencies
	$(EXEC) composer update -n
        $(EXEC) yarn upgrade
`